packages/candid/src/idl.ts

```diff
@@ -647,7 +648,7 @@ export class FixedIntClass extends PrimitiveType<bigint | number> {
   }
 
   public encodeType() {
-    const offset = Math.log2(this._bits) - 3;
+    const offset = ilog2(this._bits) - 3;
```

Comment:
Is this the right place for the fix? `this._bits` can only be `8,16,32,64`. I would expect the problem comes from taking log2 of the actual number, not `_bits`?